packages/mars/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "ethereum-mars",
   "description": "The amazing smart contract deployment tool",
   "license": "MIT",
-  "version": "0.1.6",
+  "version": "0.1.7",
```

Comment:
What about https://keepachangelog.com/en/1.0.0/ ?
That's just a little and tells much...

Reply:
closing, we have an issue for the changelog and 0.1.9 released already